### DISCUSSION TITLE
Dynamically Add/Remove input fields

### DISCUSSION THREAD

**Discussion by Fabio-Paccosi** [thedevdojo/voyager#5254] [Q&A]
Hi,

I would like to know if it was possible to dynamically insert fields each of which will fill a row of a pivot table.
The result I would like to obtain is the attached screenshot.

<img width="1386" alt="Schermata 2021-03-03 alle 15 01 02" src="https://user-images.githubusercontent.com/59596946/109817178-e90b3680-7c31-11eb-88b2-3d677ce622cb.png">

For example, by clicking on the blue button I add a row and in the 'Product' field I should be able to select elements already present on the db as in another normal relationship set on voyager.
By clicking on the 'Save' button these rows should then compose my pivot table ...

Has anyone ever implemented such a solution? Is it feasible to do this in Voyager?

Thank you very much for the attention

**Comment by emptynick** [comment]
Not an issue -> moved to discussions
